pkg/controller/controller.go

```diff
@@ -89,6 +91,25 @@ func (c *Controller) sync() error {
 	log.Println("domainMap:", domainMap)
 
 	for domainName, domain := range domainMap {
+
+		// temporary ZMS call to get the signed domain
+		signedDomain, err := zms.GetSignedDomain(domainName)
+		if err != nil {
+			log.Println("Error getting the signed domain from ZMS:", err)
+			continue
+		}
+
+		domainRBAC := m.ConvertAthenzPoliciesInToRBACModel(signedDomain.Domain)
+
+		provider := rbacv1.NewProvider()
+		rbacCRs := provider.ConvertAthenzModelIntoIstioRBAC(domainRBAC)
```

Comment:
I am thinking we need to add the validation functions calls somewhere for the specs.

Reply:
good catch. totally missed the validation. added after generating each SR/SRB spec...